src/i18n/locales/ja.json

```diff
@@ -0,0 +1,99 @@
+  "blog.authors.list.title.and": "and",
+  "components.downloadAdditional.title": "他のダウンロード",
+  "components.downloadHeader.navigation.activeSection": "ダウンロード",
+  "components.downloadHeader.navigation.title": "ダウンロード",
+  "components.downloadHeader.navigation.nodeVersion": "{lts, select, true {LATEST LTS}other {CURRENT}} VERSION {nodeVersion}",
+  "components.downloadHeader.navigation.npmVersion": "(npm を含む)",
+  "components.downloadTable.header.release": "リリース",
+  "components.downloadTable.header.status": "ステータス",
+  "components.downloadTable.header.codename": "コードネーム",
+  "components.downloadTable.header.initialRelease": "最初のリリース",
+  "components.downloadTable.header.ltsStarts": "アクティブ LTS 開始",
+  "components.downloadTable.header.maintenanceStart": "メンテナンス LTS 開始",
+  "components.downloadTable.header.endOfLife": "サポート終了",
+  "components.downloadReleases.upcomingReleases.title": "直近のリリース",
+  "components.downloadReleases.upcomingReleases.content": "Node.js の主要なバージョンは6ヶ月間カレントリリース状態になり、ライブラリの作者にサポートを追加する時間が与えられます。6ヶ月後、奇数番号のリリース(9, 11など)はサポートされなくなり、偶数番号のリリース(10, 12など)はアクティブ LTS 状態に移行し、一般利用が可能になります。LTS リリースの状態は \"長期サポート\" であり、通常は合計30ヶ月間重要なバグが修正されることが保証されています。プロダクションアプリケーションでは、アクティブ LTS またはメンテナンス LTS リリースのみを使用する必要があります。",
+  "components.downloadToggle.current": "最新",
+  "components.downloadToggle.lts": "LTS",
+  "components.downloadToggle.recommendation": "{selected, select, LTS {Recommended for most users}other {With the latest features}}",
+  "components.editLink.title": "このページを GitHub で編集",
+  "components.jsonLink.title": "JSONで閲覧する",
+  "components.footer.links.trademark": "商標ポリシー",
+  "components.footer.links.privacy": "プライバシーポリシー",
+  "components.footer.links.codeOfConduct": "行動規範",
+  "components.footer.links.security": "セキュリティ報告",
+  "components.footer.links.about": "Node.jsとは",
+  "components.footer.links.blog": "ブログ",
+  "components.footer.links.openJsFoundation": "© OpenJS Foundation",
+  "components.footer.links.github": "GitHub",
+  "components.header.links.learn": "学ぶ",
+  "components.header.links.docs": "API ドキュメント",
+  "components.header.links.download": "ダウンロード",
+  "components.hero.downloadLts": "Node.js (LTS)をダウンロード",
+  "components.hero.currentVersion": "バージョン {version}",
+  "components.hero.getCurrent": "最新({version})をダウンロード",
+  "components.hero.learn": "Node.js を学ぶ",
+  "components.installTabs.readDocs": "ドキュメントを読む",
+  "components.pagination.previous": "前へ",
+  "components.pagination.next": "次へ",
+  "components.randomContributor.thankYou": "{contributor} さんの {amount} の Node.js へのコントリビューションに感謝します",
+  "components.recentPosts.title": "最近の記事",
+  "components.searchBar.placeholder": "検索",
+  "components.searchBar.search.title": "タイプして検索",
+  "components.searchBar.search.noResults": "検索結果はありません",
+  "containers.authorList.title": "コントリビューター",
+  "containers.navigation.title": "メニュー",
+  "pages.download.description": "Download the Node.js source code, a pre-built installer for your platform, or install via a",
```

Comment:
it is normal that it is not translated

Reply:
yeah, I was struggled Japanese :( 

https://github.com/nodejs/nodejs.dev/pull/2764#discussion_r971096557

Reply:
this is not good because it is a format that depends on a specific language. it may affect other languages as well, so even if this were to be changed, it could be done separately.
https://github.com/nodejs/nodejs.dev/blob/8ac49f3837afa0e219dfa684aaa137df525d75c0/src/pages/download/index.tsx#L50-L56